external_distributions/mumps_and_scalapack/scalapack_python3_scripts/README

```diff
@@ -0,0 +1,25 @@
+Upgraded python scripts in scalapack installer to python3, using 2to3 and
+some additional tweaks applied with plenty of hand-holding from Puneet.
+When scalapack is being built these modified files overwrite the ones
+contained in the scalapack tar file.
+
+  Careful: scalapack.log is also scanned to extract some build information
+ for the subsequent mumps installation, particularly information about
+ mangling, so adding debug statements to the python scripts can have
+ unintended consequences (as I found out).
+
+  To compare the modified files against the original ones:
+
+    cd external_distributions/mumps_and_scalapack/scalapack_python3_scripts/
+    mkdir junk
+    cp scalapack_installer.tgz junk/
+    cd junk/
+    tar xvfz scalapack_installer.tgz 
```

Comment:
Just a little tip (that I only recently stumbled onto); you can just specify the location that files are untarred to with the `-C` flag. Therefore, from the outer directory, you could just call:
```bash
tar xvfz scalapack_installer.tgz -C junk/
```

Reply:
Thanks, but the code above is more transparent (Am I defensive today? Must act on at least one of your suggestions!). Let's see...